<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanConnectDialog</class>
 <widget class="QDialog" name="CanConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="ConfigurationBox">
     <property name="title">
      <string>Specify Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="errorFilterLabel">
        <property name="text">
         <string>Error Filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="errorFilterEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="loopbackLabel">
        <property name="text">
         <string>Loopback</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="loopbackCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="receiveOwnLabel">
        <property name="text">
         <string>Receive Own</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="receiveOwnCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="bitrateLabel">
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="BitRateBox" name="bitrateCombo"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="canFdLabel">
        <property name="text">
         <string>CAN FD</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="canFdCombo"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="dataBitrateLabel">
        <property name="text">
         <string>Data Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="BitRateBox" name="dataBitrateCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="CanInterfaceBox">
     <property name="title">
      <string>Specify CAN Interface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="interfaceCombo"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="interfaceRefreshButton">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="interfaceLabel">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>**PEAK CAN Converter is used</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="cancelPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitRateBox</class>
   <extends>QComboBox</extends>
   <header>bitratebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
